package.json

```diff
@@ -7,6 +7,7 @@
     "yarn": "^1.21.1"
   },
   "scripts": {
+    "prepublish": "lerna run build && gid add **/dist/*",
```

Comment:
```suggestion
    "prepublish": "lerna run build && git add **/dist/*",
```

Reply:
I think what you want is the version hook though. And not `pre`, because that's before the version is incremented.